### DISCUSSION TITLE
关于萨米肉鸽刷隐藏坍缩

### DISCUSSION THREAD

**Discussion by Morrri725** [MaaAssistantArknights/MaaAssistantArknights#11327] [Ideas]
希望可以在选择“刷坍缩”策略时，增加一个“刷到隐藏坍缩时停止”的选项，这样想要解锁二级坍缩的可以自己手动打下去，只想刷一级的可以不勾选这个选项。

**Comment by Morrri725** [comment]
（挂了好几天终于刷到了但是翻图鉴才发现只解锁了1级（悲

**Reply by Alan-Charred** [reply]
不太对劲，可以请您运行任务一轮（直到重开）后发一个Log吗？您所说的是这个功能应该有的功能，如果没有及时停下，多半是有Bug了。

**Reply by Morrri725** [reply]
不好意思我是第一次上传log，是像这样附在评论上就可以吗？
[萨米刷坍缩.log](https://github.com/user-attachments/files/18125891/default.log)

**Reply by Alan-Charred** [reply]
🤔️抱歉回复晚了，根据 Log，对坍缩范式的识别没有问题，设置也正确。
```
[2024-12-13 03:38:56.465][INF][Pxb0ef][Txd327] load_params | Expected collapsal paradigms are [目空一些, 睁眼瞎, 图像损坏, 一抹黑]
```
我来自己试一下。。。刷坍缩范式功能的原意是，当遇到需要的坍缩范式的时候，会当场停下来，所以当遇到一级隐藏坍缩范式的时候，MAA 就会停止任务，以供用户继续打到二级。

**Reply by Alan-Charred** [reply]
测试完毕，功能运作良好。在刷到想要的一级坍缩范式后，MAA 会退出当前肉鸽，但是不会重开。您可以继续探索来将这个坍缩范式加深至二级。如果您发现 MAA 的行为与我所描述的不符，请再详细描述下情况，并附上相关的 Log。感谢。

**Reply by Morrri725** [reply]
啊，那可能是我看它退出来就自己关掉了，不好意思QvQ
请问这个帖子需要删掉吗？

**Reply by Alan-Charred** [reply]
啊我来关闭这个 Issue 就好。如果遇到其他情况，欢迎再来反馈！
-------- 分割线 --------
啊原来已经被关闭了

**Comment by Daydreamer114** [comment]
cc @Alan-Charred
